ktor-client/ktor-client-tests/jvm/src/io/ktor/client/tests/utils/TestServer.kt

```diff
@@ -26,11 +29,26 @@ internal fun startServer(): Closeable {
         benchmarks()
     }.start()
 
+    val file = File("build/client-tls-test-server.jks")
+    val testKeyStore = generateCertificate(file)
```

Comment:
could you extract this in a separate method? (Like `setupTLSServer`).

Reply:
done